chapter3/chapter3.tex

```diff
@@ -248,15 +249,15 @@ \section{Stetigkeit}
   |f(q) - f(p)| \leq \frac{|f(p)|}{2}.
   \]
   Also ist $1/f$ auf der Menge $(p-a, p+a) \cap A$ definiert,
-  da dort $|f(q)| > 0$.
+  da dort $|f(q)| \geq \frac{|f(p)|}{2} > 0$. % braucht man für Beweis
   Sei nun $\varepsilon > 0$. 
   Berechne
    \begin{align*}
      \left| \frac{1}{f(q)} - \frac{1}{f(p)} \right| 
      & = \left| \frac{f(p) - f(q)}{f(q)f(p)} \right| \\
      & \leq 2 \cdot \frac{|f(p) - f(q)|}{|f(p)|^2}. 
   \end{align*}
-  Wähle $\delta > 0$, so dass $\delta \leq a$ 
+  Wähle $\delta > 0$ so, dass $\delta \leq a$ 
```

Comment:
Ist glaube ich nicht konsistent mit den unteren "so dass" Kommas, oder?

Reply:
Eigentlich empfiehlt der Duden eh, in der Regel "sodass" in einem Wort zu schreiben :) Ich hatte mal im Kopf, dass "sodass" einen Konsekutivsatz einleitet ("Es gibt ein bestimmes x, sodass blabla" <=> "Weil es ein bestimmtes x gibt, blabla"), und dass "... so (...), dass" das Verb modifiziert ("Wähle x so, dass" <=> "Wähle x auf eine solche Art und Weise, dass"). Aber vermutlich sind das Spitzfindigkeiten, die Bastian Sick Geld aufliefern und sonst nix bringen.

Reply:
Und beim "sodass" in einem Wort käme dann das Komma davor?

Reply:
Ah nein, ich habe das wohl falsch verstanden. Meistens ist bei mir "so, dass" gemeint: Ich wähle ein x, welches eine bestimmte Eigenschaft erfüllt. Das wäre nicht konsekutiv, oder?